src/views/InvolvementProfile/InvolvementProfile.module.scss

```diff
@@ -11,3 +11,22 @@
 .update_image_components {
   margin-top: 15px;
 }
+
+.removeImage {
+  background-color: var(--mui-palette-error-main);
+  color: var(--mui-palette-secondary-contrastText);
+}
+
+.dialogContentText {
+  text-align: 'center';
+}
+
+.imageDrop {
+  max-width: 320px;
+  max-height: 320px;
+  border-radius: 0.5rem;
+}
+
+.activityURL {
+  font-weight: bold;
```

Comment:
is there a reason some of your styles are in `snake_case` then others in `camelCase`?

Reply:
> I don't know about @EjPlatzer but since this project is meant to refactor effectively all `css` modules, we might as well find a way to handle naming properly.

This project definitely is not meant to change all CSS across the whole site. The scope of the project is really just to make everything use the mui CSS vars correctly, but we've decided to move all styles to SCSS files that wasn't already in a file since those styles often need to be deeply updated anyways.

Also, I simply don't think it matters that much how styles are cased, and it doesn't seem like a good use of time and effort to scour our files and simply rename a bunch of classes just for the sake of renaming them. My one rule is that class names should be kept as simple as is sensible, especially in light of the fact that they're scoped and so don't need to be overly specific.

> BEM has solid documentation which makes sense: https://en.bem.info/methodology/naming-convention/

BEM is one of many competing modes of thought on how to organize a massive amount of style and component primitives. I don't think it has as much value for our use case. Firstly, because we're using CSS modules so our class names don't need to be globally unique nor consistent and predictable enough to be reused in many disparate contexts. Our class names really only need to make sense in the context of one specific component and it's particular needs. Secondly, we aren't building a component library from scratch, so we don't need to and shouldn't have styles for every block and element and modifier (going be the BEM terminology). We are basically only writing styles to occasionally override and customize the existing MUI styles.

Reply:
All that to say, I think the rule for our class names should be "Keep it short and descriptive". Also, don't use hyphens because they don't work in JSX.

Reply:
Gotcha, I think for the most part we've loosely followed that rule, I'll emphasize it to my teammates, but agreed, we shouldn't change the names in all the files just for the sake of it...